runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -361,6 +363,46 @@ NNFW_STATUS nnfw_session::prepare()
   return NNFW_STATUS_NO_ERROR;
 }
 
+NNFW_STATUS nnfw_session::prepare_pipeline(const char *map_file_path)
```

Comment:
how about using predefined map_file_path to remove the argument ? For example, map_file_path is provided as a config file just like configs file in nnpackage spec.

```
{
    "major-version" : "1",
    "minor-version" : "1",
    "patch-version" : "0",
    "configs"     : [ "model.cfg" ],
    "partition_map" : [ "partition.json" ],
    "models"      : [ "mymodel.model" ],
    "model-types" : [ "tflite"]
}
```

Reply:
If the user explicitly set the map_file_path, runtime will use the map_file_path, but If the user does not use the map_file_path, runtime will try to find the map_file in nnpackage directory

Reply:
If the policy supports only the map file in the nnpackage, I agree with you. otherwise we have to consider to support the external path.